### PR TITLE
ADCM-2454 Add tests to role_spec file

### PR DESCRIPTION
**Pull request by acmnu**

### CODE REVIEW

**Review by a-alferov — APPROVED** [review]

File: python/rbac/upgrade/role_spec.yaml [hunk 1]
```diff
@@ -456,7 +496,7 @@ roles:
               codenames:
                 - delete
 
-    - name: Map hosts
+    - name: Add hosts to the Cluste
```
Comment (kuhella) [inline python/rbac/upgrade/role_spec.yaml:499]: Cluster

Comment (acmnu) [reply python/rbac/upgrade/role_spec.yaml]: Yep, fixed.

File: python/rbac/upgrade/role_spec.yaml [hunk 2]
```diff
@@ -1059,11 +1086,28 @@ roles:
...
+        - Upgrade application bundle
         - Create host
         - Upload bundle
         - Remove bundle
 
+    - name: Provider Administrator
+      type: role
+      parametrized_by:
+        - provider
+        - hosts
```
Comment (a-alferov) [inline python/rbac/upgrade/role_spec.yaml:1098]: fix pls on `host`

Comment (a-alferov) [reply python/rbac/upgrade/role_spec.yaml]: I fixed

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Pull request merged**